.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2.1
+orbs:
+  # use Cypress orb from CircleCI registry
+  cypress: cypress-io/cypress@1.8.0
+working_directory: ~/repo
+jobs:
+  cypress/run:
+    docker:
+    - image: cypress/base:10
+    parallelism: 2
+    steps:
+    - checkout
+    - restore_cache:
+        keys:
+        - cache-{{ .Branch }}-{{ checksum "package.json" }}
+    - save_cache:
+        key: cache-{{ .Branch }}-{{ checksum "package.json" }}
+        paths:
+        - ~/.npm
+        - ~/.cache
+    - attach_workspace:
+        at: ~/
+    - run:
+        name: Run Cypress tests
+        command: npm run cy:ci
+
+workflows:
+  build:
+    jobs:
+      # split into install + test jobs when Mac caching is correct
```

Comment:
why?

Reply:
I can't see the install job. Does CircleCI do this by default when the Cypress orb is used?

Reply:
cypress orb takes care of that